prow_config.yaml

```diff
@@ -63,6 +63,25 @@ workflows:
       workflowName: kfctl-go
       useBasicAuth: false
       useIstio: true
+  # Run a nightly build of kfctl and push the binary to GCS    
+  - app_dir: kubeflow/kubeflow/testing/workflows
```

Comment:
Why do we need a new prow job? It looks like we are copying the binary in kfctl_go_test.py.
That gets executed on every pre/post and periodic run.
Isn't that sufficient?